packages/composables/src/useProductAssociations.ts

```diff
@@ -53,16 +53,16 @@ export function useProductAssociations(
     searchParams: ShopwareSearchParams;
   }) => {
     isLoading.value = true;
-    const method = params.method || "get";
+    const method = params?.method || "get";
     try {
       if (method && method === "get") {
         const response = await invokeGet(
           {
             address: `${getProductDetailsEndpoint(
-              product.value.id
-            )}/${association}${params.searchParams || ""}`,
+              product.value?.id,
```

Comment:
The whole function should not be invoked if the product does not exist. Maybe check it at the beginning and throw an error?
What's in response from API at this point?

Reply:
we will get 400 error due to invalid uuid. I will revert this line and throw an error on missing product object